src/iop/highlights.c

```diff
@@ -71,12 +71,12 @@ static float highlights_clip_magics[6] = { 1.0f, 1.0f, 0.987f, 0.995f, 0.987f, 0
 
 typedef enum dt_iop_highlights_mode_t
 {
+  DT_IOP_HIGHLIGHTS_OPPOSED = 5,  // $DESCRIPTION: "inpaint opposed"
```

Comment:
I would keep the order here. The inpaint opposed is the last implemented and the order here has not correlation with the order in the menu.

Reply:
I'll prefer having that before merging. This is ready to me, are you planing more on this PR?

Reply:
No, I think the highlights stuff if finished for now. We will continue our work on possibly better maths though 😎